apps/studio/components/interfaces/Database/Extensions/EnableExtensionModal.tsx

```diff
@@ -165,13 +184,14 @@ const EnableExtensionModal = ({ visible, extension, onCancel }: EnableExtensionM
                     name="schema"
                     value={defaultSchema}
                     label="Select a schema to enable the extension for"
-                    descriptionText={`Extension must be installed in ${defaultSchema}.`}
+                    descriptionText={getSchemaDescriptionText(extension.name, defaultSchema)}
                   />
                 ) : (
                   <Listbox
                     size="small"
                     name="schema"
                     label="Select a schema to enable the extension for"
+                    descriptionText={getSchemaDescriptionText(extension.name, null)}
```

Comment:
I’ve made a reusable function `getSchemaDescriptionText` because an inline ternary is hard to read and not extensible for when we (should) add more recommended schemas in the future.